lib/i18n/tasks/scanners/ruby_ast_scanner.rb

```diff
@@ -131,7 +132,7 @@ def setup_matchers
             message: message,
             scanner: self
           )
-        end
+        end << AstMatchers::RailsModelMatcher.new(scanner: self)
```

Comment:
@glebm Should we check some config if the rails-matcher should be used?

Reply:
Yeah, let's think of a way to make this configurable